docs/sample_config.yaml

```diff
@@ -2402,3 +2415,40 @@ opentracing:
     #
     #  logging:
     #    false
+
+
+## Workers ##
+
+# When using workers this should be a map from `worker_name` to the
+# HTTP replication listener of the worker, if configured.
+#
+#instance_map:
+#  worker1:
+#    host: localhost
+#    port: 8034
+
+# Experimental: When using workers you can define which worker should
+# handle event persistence or typing notifications. Any worker
+# specified here must also be in the `instance_map`.
+#
+#stream_writers:
+#  events: worker1
+#  typing: worker1
+
+
+# Configuration for Redis when using workers.
+#
+redis:
+  # Uncomment the below to enable using Redis to replicate data between workers.
```

Comment:
what happens if it's not enabled?

Reply:
Hmm, I guess we should reword this to make it clear that it needs (unless using old style replication) to be enabled if using workers